code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -160,6 +160,7 @@ Please contact me on #coderbus IRC. ~Carn x
 		overlays += (new_overlays - old_overlays)
 		overlays -= (old_overlays - new_overlays)
 		cached_standing_overlays = new_overlays
+		overlays |= eyes_shine() //Only applies eyeshine if the eyes are uncovered and have significantly high darksight or XRAY vision.

```

Comment:
This is an absolutely terrible way to do this, you should use what's already there and apply/remove the overlays correctly. This also means that the eyes_shine() proc should return a boolean value, not an image.

Reply:
The image's .layer needs to be lighting_layer + 1. This needed to dodge the above code else it'd get shuffled into a different layer. There is no code that currently accomplishes this but I'll keep looking. It's clear I'll have to be a bit more creative.